ansible/roles/deploy-apigee-proxy/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+# NOTE: uri isn't used here because it exhibits some odd behaviour AND it doesn't return the request body
+- name: upload proxy bundle
+  shell:
+    cmd: "curl --fail -X POST -H 'Authorization: Bearer {{ APIGEE_ACCESS_TOKEN }}' -F 'file=@/tmp/proxy.zip' -H 'Content-Type: multipart/form-data' {{ apis_api_uri }}\\?action\\=import\\&name\\={{ SERVICE_NAME }}-{{ APIGEE_ENVIRONMENT }}"
```

Comment:
could use ansible temp_dir / temp_file module to avoid file clash ...  ( or ensure absent )